package.json

```diff
@@ -80,6 +80,6 @@
     "sinon": "^1.15.4"
   },
   "peerDependencies": {
-    "react": "0.13.x || 0.14.x || 15.* || 15.0.0-rc.2"
+    "react": "0.13.x || 0.14.x || 15.* || ^15.0.0-rc"
```

Comment:
semver ranges don't actually match against prerelease versions, so this is not a good change.